# Open Access Datasets

This is a collection of openly accessible datasets, as reported by the researchers. This collection contains both datasets with both thick and thin image datasets and those with thick and thin image datasets only.


## Datasets
- [Key Learning Outcomes](#key-learning-outcomes)
- [Main Learning Outcomes](#main-learning-outcomes)
- [Thick Blood Smear Image Datasets](https://github.com/ItunuIsewon/Malaria_Blood_Smear_Images/blob/main/Open_Access/Thick-OA.md)
- [Thin Blood Smear Image Datasets](https://github.com/ItunuIsewon/Malaria_Blood_Smear_Images/blob/main/Open_Access/Thin-OA.md)





ADEGOKE FAITH OMOLARA PERSONAL STATEMENT - Wellcome Trust DTP in Genomic Epidemiology and Public Health Genomics
Biology has been a captivating subject for me since high school, sparking my passion to pursue it as a career. With this strong interest, I studied Science Laboratory Technology with a major in Biological Science/Biotechnology, for my bachelor’s degree. The 5-year study’s curriculum ranged from developmental stages of organisms to genetics and many more giving me exposure to a wide range of biological subject matter. For my research, I worked on the “Production and characterization of Lipase produced by Pseudomonas aeruginosa isolated from oil mill effluents”. From the research, it was posited that the microorganism is a microbial source of lipase and a good candidate for industrial applications. I gained skills in microbiology techniques, biochemical analysis, scientific writing, and presentations. I finished with a second-class upper grade (CGPA: 4.1/5.0). With this same passion, I took up internship positions in a medical laboratory and then in a biological science laboratory where I was taught laboratory safety and I also assisted in biology practical classes for undergraduates and a few projects. One of the projects was to do a microbial analysis of the water supply in the different student residences at the university. From the research, the hostels with a high load of microbes in their water were investigated and recommendations were made for quick intervention and the right management.
My research journey continued with a master's degree in bioinformatics, sponsored by the World Bank (CGPA: 4.76/5.0). During my master's, there was an emphasis on the need to bridge the gap between biological science and computation since there has been an increase in the generation of biological data. From my course work, I gained knowledge in Data Structures, programming with Python, C#, and other computational biology courses and for my research work, I developed an automated malaria detection system using deep learning models. All available image datasets were curated, and then the system was able to predict the infection status, type of species, type of smear, stain type, and the optical train (equipment) used in capturing the images. The aim of the research was to solve the problem of some image datasets lacking this information, which is vital in determining the methods and techniques to be used for research. I also developed a web application prototype using the Python Flask micro-framework as proof of concept. 
The result of my project emphasized the significance of automated diagnosis in the diagnosis and treatment of diseases. Also, studying a disease from both genomic and imaging viewpoints would have higher prospects of giving an accurate picture of the underlying mechanisms influencing the disease. Hence, my interest is to utilize my doctoral research to combine imaging and genomics in solving biological problems. My master’s degree experience has given me skills in Python programming, machine learning, Git, R, Linux, gene expression analysis and genome-wide association study (GWAS) analysis.
In addition to the technical skills, I have honed my presentation skills which help me to convey my ideas and results to others effectively. I also had leadership experience serving as my class representative throughout the two-year master's program. I can work effectively in a team as I believe in what Helen Keller wisely said, "Alone, we can do so little; together, we can do so much." Collaboration is very important and useful in driving advancements in research. I am highly motivated and resilient.
My choice of applying for the PhD program in Genomic Epidemiology and Public Health Genomics is because the program produces world-class and excellent students with its skills development both for research projects and one’s career. The opportunity to also work with industry partners is a motivating factor. I am confident that this program will fulfil my desire to get a quality doctoral education with its good faculty and research facilities. I believe that I have the ability and skill to excel as an outstanding doctoral student.
Ultimately, it is my strong desire to attain a faculty position at a university and to be involved in continuous research in bioinformatics helping me to make significant contributions to my immediate environment and to the scientific community at large.
Thank you for considering my candidacy.


><details>
> <summary>## Key Learning Outcomes</summary>     
>
> After completing this practical, you should be able to:
>  1. know how to adjust for ascertainment bias in case-control analysis
>  2. Know how over-fitting aﬀects PRS results and how to handle it 
>  3. understand distribution of PRS
>  4. understand diﬀerent file formats involved in gene-set analysis
>  5. understand diﬀerence between self-contained and competitive gene-set analyses
>  6. Calculate pathway based PRS      
>
></details>
>
> 
---



## Main Learning Outcomes
After completing this practical, you should be able to:
  1. know how to adjust for ascertainment bias in case-control analysis
  2. Know how over-fitting aﬀects PRS results and how to handle it 
  3. understand distribution of PRS
  4. understand diﬀerent file formats involved in gene-set analysis
  5. understand diﬀerence between self-contained and competitive gene-set analyses
  6. Calculate pathway based PRS
